Src/Extern/SUNDIALS/AMReX_SundialsIntegrator.H

```diff
@@ -146,14 +155,15 @@ private:
         NLS = nullptr;            /* empty nonlinear solver object */
         LS = nullptr;             /* empty linear solver object    */
         arkode_mem = nullptr;     /* empty ARKode memory structure */
+        cvode_mem = nullptr;      /* empty CVODE memory structure  */
         NLSf = nullptr;           /* empty nonlinear solver object */
         LSf = nullptr;            /* empty linear solver object    */
         inner_mem = nullptr;      /* empty ARKode memory structure */
         mristep_mem = nullptr;    /* empty ARKode memory structure */
 
         // Arbitrary tolerances
-        reltol          = 1e-4;
-        abstol          = 1e-4;
```

Comment:
Other codes might rely on the default values here, even though the comment says it's arbitrary.